lib/cc/analyzer/issue_validations.rb

```diff
@@ -0,0 +1,25 @@
+module CC
+  module Analyzer
+    module IssueValidations
+      autoload :CategoryValidation, "cc/analyzer/issue_validations/category_validation"
+      autoload :CheckNamePresenceValidation, "cc/analyzer/issue_validations/check_name_presence_validation"
+      autoload :DescriptionPresenceValidation, "cc/analyzer/issue_validations/description_presence_validation"
+      autoload :LocationFormatValidation, "cc/analyzer/issue_validations/location_format_validation"
+      autoload :OtherLocationsFormatValidation, "cc/analyzer/issue_validations/other_locations_format_validation"
+      autoload :PathExistenceValidation, "cc/analyzer/issue_validations/path_existence_validation"
+      autoload :PathIsFileValidation, "cc/analyzer/issue_validations/path_is_file_validation"
+      autoload :PathPresenceValidation, "cc/analyzer/issue_validations/path_presence_validation"
+      autoload :RelativePathValidation, "cc/analyzer/issue_validations/relative_path_validation"
+      autoload :TypeValidation, "cc/analyzer/issue_validations/type_validation"
+      autoload :Validation, "cc/analyzer/issue_validations/validation"
+
+      def self.validations
+        constants.sort.map do |const_name|
+          const_get(const_name)
+        end.select do |klass|
```

Comment:
I do try and avoid it because I think it looks a bit off. In this case, I'd be open to `constants.sort.map(&method(:const_get)).select do |klass|`. Just an idea, I'd also be find disabling the rule if that's how everyone feels.